en/deploy/README.md

```diff
@@ -111,7 +111,7 @@ Type the following into your console (replace `<your-github-username>` with the
 {% filename %}command-line{% endfilename %}
 ```
 $ git remote add origin https://github.com/<your-github-username>/my-first-blog.git
-$ git push -u origin master
+$ git push origin main
```

Comment:
Why did you drop `-u`? The tracking is quite useful.

Reply:
Fixed it